### DISCUSSION TITLE
内置通过Telegram Bot发消息的功能

### DISCUSSION THREAD

**Discussion by athurg** [usememos/memos#1676] [Ideas]
内置对telegram bot的支持后，可以通过直接在telegram里向这个机器人发消息，来发新的Memos。

当然，也可以在此基础上扩展出一些新的机制，比如别的用户有开放的memos发出来的时候，通过机器人提醒（通知）。

实现方案上，有两种方案（可以都支持）：
- 自建方案：自己创建并配置机器人的token，只能自己用。
- 托管方案：配置一个公共的机器人（及其后台），每个人向这个机器人发送带认证信息的memos地址。这样机器人收到消息后，取出你的配置信息并发memos。

**Comment by cxplay** [comment]
https://github.com/qazxcdswe123/telegramMemoBot

**Reply by athurg** [reply]
这个东西我看过。不过我主要在想的是，能否让最终用户用起来更简单。

比如不用额外跑什么服务，只需要配置一下必要的telegram机器人信息就可以。

**Comment by dianso** [comment]
不错，希望支持

**Comment by athurg** [comment]
已完成。 参见： #1740 #1747 #1764
